scripts/load_org_data_to_hive_and_transform_it.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
it's always better to have a separate load and transform the flow 
bad naming again. what if you will have another transformation